src/simulation/simulation_internal.jl

```diff
@@ -16,25 +15,33 @@ mutable struct SimulationInternal
     recorders::Vector{Symbol}
     console_level::Base.CoreLogging.LogLevel
     file_level::Base.CoreLogging.LogLevel
+    #cache_size_mib::Int
```

Comment:
@daniel-thom I think these parameters should be defined here to be used later in the build process.

Reply:
Save this values here so they are set at build time for consistency.